auth-web/src/components/auth/Signin.vue

```diff
@@ -52,6 +52,12 @@ export default class Signin extends Mixins(NextPageMixin) {
         if (this.idpHint === 'bcsc') {
           await this.syncUserProfile()
           await this.syncOrganizations()
+          // eslint-disable-next-line no-console
+          console.info('[SignIn.vue]Logged in User.Starting refreshTimer')
+          var self = this
+          this.$tokenService.initUsingUrl(`/${process.env.VUE_APP_PATH}/config/kc/keycloak.json`).then(function (success) {
```

Comment:
My preference is using `async/await` instead for asynchronous.  Do you have any objections if we stick to that pattern?

Reply:
Will change it

Reply:
just the habit :-( .. changing it